src/main/scala/Day3.sc

```diff
@@ -0,0 +1,122 @@
+//FUNCTIONAL 1
+val countryCapital = java.util.TimeZone.getAvailableIDs()
+
+countryCapital.map(_.split('/').toList.drop(1).mkString).filter(_.length>1)
+
+//FUNCTIONAL 2
+def fibonacci(intList: List[Int]):Boolean = {
+  var result:Boolean = true
+  val sortList = intList.sorted
+  for(i<- 0 to sortList.length -3)
+    if (sortList(i) + sortList(i + 1) == sortList(i + 2)) result = true
+    else result = false
+  result
+}
+
+fibonacci(List(1,1,2,3,5,8))
+
+//Is it a factorial
+def factorial(n: Int): Int = {
+  if (n == 0) 1
+  else n * factorial(n-1)
+}
+
+def isAFactorial(number:Int, counter:Int = 1): Any= {
+  if(factorial(counter) == number) s"$counter!"
+  else if (factorial(counter) < number) isAFactorial(number, counter+1)
+  else if (factorial(counter) > number) "NONE"
+}
+
+isAFactorial(3628800)
+
+//CIPHER
+val alphabet = "abcdefghijklmnopqrstuvwxyz"
+val alphabetArray = alphabet.toCharArray
```

Comment:
why not **val alphabetArray = "abcdefghijklmnopqrstuvwxyz".toCharArray** or **alphabetArray = ('a' to 'z').toArray**